src/Client/Components/Pages/Community.razor

```diff
@@ -41,10 +21,6 @@
 					<div class=""><Href href="https://github.com/SteeltoeOSS" NewWindow="true">Steeltoe on GitHub</Href></div>
 					<div>Find the code for Steeltoe projects, raise issues, suggest changes, and view milestones and releases.</div>
 				</div>
-				<div class="my-5">
-					<div class=""><Href href="https://twitter.com/steeltoeoss" NewWindow="true">Steeltoe on Twitter</Href></div>
-					<div>Stay up to date with the latest Steeltoe releases and blog posts.</div>
-				</div>

```

Comment:
The channel has 888 followers, don't we want to keep this?

Reply:
Removed because I don't really want to actively promote it when we barely use it... but we have still at least been publishing release notices there, so I suppose it's worth keeping for now